package.json

```diff
@@ -108,7 +107,7 @@
     "native-is-elevated": "0.7.0",
     "native-keymap": "^3.3.5",
     "native-watchdog": "^1.4.1",
-    "node-pty": "1.1.0-beta35",
+    "node-pty": "1.1.0-beta38",
```

Comment:
Need to also update remote/package.json for node-pty as it's shipped with the server.

Reply:
Silly question.. I just realized for remote/ package-lock.json, I had to manually edit the version number.

Usually `npm install` would update package-lock.json . But for remote is it manual like this?